keanu-project/src/main/java/io/improbable/keanu/vertices/ConstantVertex.java

```diff
@@ -10,10 +10,7 @@
 import io.improbable.keanu.vertices.generic.nonprobabilistic.ConstantGenericVertex;
 import io.improbable.keanu.vertices.intgr.nonprobabilistic.ConstantIntegerVertex;
 
-public class ConstantVertex {
-
-    private ConstantVertex() {
-    }
+public interface ConstantVertex {
```

Comment:
The methods no longer need to be explicitly called public now that it's an interface.

Reply:
Fixed